docs/tutorials/mpv.md

```diff
@@ -384,7 +384,7 @@ k cycle_values sub-ass-override "force" "no"
 
 Auto profiles allow users to automate actions based on certain conditions. Tasks such as [debanding](#debanding) and [subtitle restyling](#subtitle-restyling) can be applied automatically when these conditions are met, such as file names.
 
-For instance, some seasonal releases may exhibit banding issues and less-appealing subtitles. To address this, we can create a `simulcast` auto profile, which will automatically apply debanding and restyle the subtitles when the filename includes specific keywords, such as the group tag.
+For instance, some seasonal releases may exhibit banding issues and use subjectively less-appealing subtitle fonts. To address this, we can create a `simulcast` auto profile, which automatically applies [debanding](#debanding) and [subtitle restyling](#subtitle-restyling) when playing releases created by SubsPlease, Erai-raws, Tsundere-Raws, VARYG, or HorribleSubs.
```

Comment:
uploaded by instead of created by

Reply:
Resolved in https://github.com/Snaacky/thewiki/pull/150/commits/feee238e9da3cda5e2dee995a8d6e18f71366fab.